### ISSUE TITLE
objects expKO and expWT for script cohesinKO.data.R

### ISSUE THREAD

**Issue by dpolychr** [ibn-salem/cohesinKO#4]
Hi Jonas,

Concerning lines 56 and 59 where you have the objects expKO and expWT:

I guess the aim here is to get the expression df for the KO and the WT respectively, would that be correct?

Running the first command at line 56, it gives both the WT and the KO columns (FL) though

Maybe do something like:

expKO <- expDF %>% select(1:2, contains("FL"))

expWT <- expDF %>% select(1:2, contains("WT"))

**Comment by ibn-salem** [comment]
Hi Dimitirs, 
thanks for reporting this. Yes, as you say, this was supposed to get only the expression in WT or KO conditions respectively. But does not work correctly. This code was inserted in commit https://github.com/ibn-salem/cohesinKO/commit/b247a5de5e0fa4c6a5501229556b91cfcb9da084 where I played with tidyverse packages to clean things up but never finished. 
I checked and this error was not before my report on correlation analysis (report v03) and does not affect the differential expression analysis, since the two objects `expKO` and `expWT` are not used to build the `tidyGeneDE` table.
But I will fix this with your suggested code. Thank you. Note that also other downstream analysis using the correlation data might not work in the moment.
